NachoSpigot-Server/src/main/java/net/minecraft/server/MathHelper.java

```diff
@@ -295,9 +306,9 @@ public static double i(double var0) {
         f = new double[257];
 
         for(var0 = 0; var0 < 257; ++var0) {
-            double var1 = (double)var0 / 256.0D;
-            double var3 = Math.asin(var1);
-            f[var0] = Math.cos(var3);
+            double var1 = (double) var0 / 256.0D;
+            double var3 = (fastMathMode == true ? (FastMath.asin(var1)) : (Math.asin(var1)));
+            f[var0] = (fastMathMode == true ? (FastMath.cosQuick(var3)) : (Math.cos(var3)));
```

Comment:
```suggestion
            f[var0] = Math.cos(var3);
```

The cosQuick method has even worse results than the cos method